docs/en/enterprise-edition/policy-reference/secrets-policies/secrets-policy-index/git-secrets-115.adoc

```diff
@@ -0,0 +1,49 @@
+== Azure Machine Learning web service API key detected in code
+
+
+=== Policy Details
+
+[width=45%]
+[cols="1,1"]
+|===
+|Prisma Cloud Policy ID
+|TBD
+
+|Checkov Check ID
+|CKV_SECRET_115
+
+|Severity
+|MEDIUM
+
+|Subtype
+|Build
+
+|Frameworks
+|Git
+
+|===
+
+
+=== Description
+
+An Azure Machine Learning web service API key is essential for accessing and utilizing Azure Machine Learning web services. This key is critical for authenticating requests to the service and ensuring that only authorized applications can interact with it. Exposure of this key can lead to unauthorized access, misuse of the API, and potential data breaches. Securely managing this key is crucial to maintaining the security and integrity of your applications. Regularly reviewing and updating your security settings and keys is important to prevent unauthorized usage. For detailed security guidelines and best practices, refer to the latest https://learn.microsoft.com/en-us/purview/sit-defn-azure-machine-learning-web-service-api-key[Azure Machine Learning documentation and support] resources.
```

Comment:
The Azure Machine Learning web service API Key serves as an essential credential for accessing Azure Machine Learning web services. This key acts as a security measure, guaranteeing that only authorized applications can interact with the API and utilize the services provided. Exposing the API key can lead to unauthorized access, misuse of the API, and potential data leaks. For this reason, it's crucial to manage your Azure Machine Learning web service API Key securely and regularly review your security practices to prevent any unauthorized usage.

For comprehensive security guidelines and best practices regarding Azure Machine Learning web service API Key management, refer to the latest documentation and support resources available at  https://learn.microsoft.com/en-us/purview/sit-defn-azure-machine-learning-web-service-api-key[Azure Machine Learning documentation and support].